life/modernized/02-life-of-gilbert-latey.adoc

```diff
@@ -1636,6 +1682,10 @@ In the year 1694, being the fourth of the reign of King William and Queen Mary,
 Gilbert attended the king at Kensington, with the following petition,
 which had so good effect, that one hundred Friends were discharged.
 
+[.embedded-content-document.address]
```

Comment:
The number I wrote down for this doesn't match up to where it is, so I"m not entirely sure I"m talking about this letter, but the beginning has the weird rendering that I know we've dealt with before, but I wasn't sure what to do exactly.